### PR TITLE
chore(admin): remove react-helmet

### PR DESCRIPTION
**Pull request by christiancp100**
### What does it do?

It removes the react-helmet dependency from all the packages

### Why is it needed?

React Helmet adds unnecessary weight to Strapi

### How to test it?
Test that the titles work as they were working before

### Related issue(s)/PR(s)

* resolves DX-1252

### CODE REVIEW

File: packages/core/admin/admin/src/components/PageHelpers.tsx [hunk 1]
```diff
@@ -231,13 +231,29 @@ const Protect = ({ permissions = [], children }: ProtectProps) => {
...
 };
 
+/* -------------------------------------------------------------------------------------------------
+ * Head Title
+ * -----------------------------------------------------------------------------------------------*/
+export interface TitleProps {
+  children: string;
+}
+
+const Title = ({ children: title }: TitleProps) => {
```
Comment (joshuaellis) [inline packages/core/admin/admin/src/components/PageHelpers.tsx:241]: This could do with some JSDoc since it's public 👍🏼

**Review by joshuaellis — APPROVED** [review]
Great work ✅

File: packages/core/admin/admin/src/components/PageHelpers.tsx (hunk 1 above)
Comment (joshuaellis) [reply packages/core/admin/admin/src/components/PageHelpers.tsx:246]: ^^ just for future reference.

File: packages/core/content-type-builder/admin/src/pages/App/index.tsx [hunk 2]
```diff
@@ -34,9 +33,10 @@ const App = () => {
     }
   }, []);
 
+  // FIXME Error here
```
Comment (jhoward1994) [inline packages/core/content-type-builder/admin/src/pages/App/index.tsx:36]: Can we remove this?

**Review by jhoward1994 — APPROVED** [review]

File: packages/core/content-type-builder/admin/src/pages/App/index.tsx (hunk 2 above)
Comment (joshuaellis) [reply packages/core/content-type-builder/admin/src/pages/App/index.tsx:36]: .... no, I think we probs need to to remember that this page, is less than desirable and needs some love.

**Pull request merged**